plugins/msg-broker/mcollective/lib/openshift/mcollective_application_container_proxy.rb

```diff
@@ -34,7 +34,9 @@ class MCollectiveApplicationContainerProxy < OpenShift::ApplicationContainerProx
     # * id: string - a unique app identifier
     # * district: <type> - a classifier for app placement
     #
-    def initialize(id, district=nil)
+    def initialize(id, district=nil, blocks_multiplier=Rails.configuration.msg_broker[:quota_blocks_buffer], inodes_multiplier=Rails.configuration.msg_broker[:quota_inodes_buffer])
```

Comment:
Not an actual issue, but style-wise this is probably incorrect. The line is far too long, perhaps set blocks_multiplier et al to `nil` by default and set it to the configuration value on the next line if it is `nil`.